apps/meteor/app/api/server/v1/users.ts

```diff
@@ -104,10 +103,6 @@ API.v1.addRoute(
 
 			await saveUser(this.userId, userData);
 
-			if (this.bodyParams.data.customFields) {
```

Comment:
This will cause a change in behavior: if there's any issue with the custom fields, this endpoint will no longer save anything to the user. With the old code it would save the changes to the user core data and only throw an error afterwards.

I don't know which behavior we want, but it might be a breaking change.